tests/tests/txpool_api.rs

```diff
@@ -89,21 +96,7 @@ async fn test_txpool_content(#[future] katana: Katana, _setup: ()) {
 #[awt]
 #[tokio::test(flavor = "multi_thread")]
 async fn test_txpool_content_from(#[future] katana: Katana, _setup: ()) {
-    // Start the Kakarot RPC server and retrieve the server address and handle
-    let (server_addr, server_handle) =
-        start_kakarot_rpc_server(&katana).await.expect("Error setting up Kakarot RPC server");
-
-    // Generate a vector of random bytes
-    let bytes: Vec<u8> = (0..RANDOM_BYTES_SIZE).map(|_| rand::random()).collect();
-    let mut unstructured = arbitrary::Unstructured::new(&bytes);
-
-    // Generate 10 pending transactions and add them to the database
-    let mut pending_transactions = Vec::new();
-    for _ in 0..10 {
-        pending_transactions
-            .push(StoredPendingTransaction::arbitrary_with_optional_fields(&mut unstructured).unwrap().tx);
-    }
-    katana.add_pending_transactions_to_database(pending_transactions).await;
+    let (server_addr, server_handle, katana) = initial_setup(katana).await;
```

Comment:
You could check after this line the content of your pending pool by printing something like

```rust
katana
        .eth_provider()
        .database()
        .get_one::<StoredPendingTransaction>(None, None)
 ```
 
to check it is correct after the function but I'm not sure about it